doc/lua_api.md

```diff
@@ -5780,6 +5780,8 @@ Utilities
       particle_blend_clip = true,
       -- The `match_meta` optional parameter is available for `InvRef:remove_item()` (5.12.0)
       remove_item_match_meta = true,
+      -- The HTTP API supports the HEAD and PATCH methods (5.12.0)
+      httpfetch_additional_methods = true,
```

Comment:
And when there's a new method, will the field be called `httpfetch_additional_methods_new_additionals`?
Jokes aside, it might be more suitable to name it `httpfetch_have_HEAD_PATCH = true` or similar.

Reply:
If we decide the update the API again it's likely that we would make `method` a fully custom field, so the feature flag would be different (and also the last time this feature would be touched).
But I can change it if you want.

Reply:
mhm okay. Whatever. Fine by me.